src/details/ArborX_Ray.hpp

```diff
@@ -204,15 +204,20 @@ bool intersects(Ray const &ray, Box const &box)
 KOKKOS_INLINE_FUNCTION int findLargestComp(Vector const &dir)
 {
   int kz = 0;
+
+  auto max = std::abs(dir[0]);
+
   for (int i = 1; i < 3; i++)
   {
-    float compmax = std::fabs(dir[i - 1]);
-    if (std::fabs(dir[i]) > compmax)
+    auto f = std::fabs(dir[i]);
```

Comment:
We can't use namespace `std` in code that runs on the device portably.

Reply:
To be clear we do not request this to be changed